src/schema/submission_centric_schema.yaml

```
@@ -210,6 +211,8 @@ classes:
           One or more attributes that further characterizes this Library Preparation Protocol.
       target regions:
         required: false
+        multivalued: true
+        inlined_as_list: true
```

Don't you have to set both inlined and inlined_as_list as in this example https://linkml.io/linkml/schemas/inlining.html#inlining-as-a-list?

```suggestion
        multivalued: true
        inlined: true
        inlined_as_list: true
```